py_zipkin/zipkin.py

```diff
@@ -143,6 +154,20 @@ def __init__(
         if self.sample_rate is not None and not (0.0 <= self.sample_rate <= 100.0):
             raise ZipkinError('Sample rate must be between 0.0 and 100.0')
 
+        if not set(include).issubset(set(STANDARD_ANNOTATIONS.keys())):
+            raise ZipkinError(
+                'Only %s are supported as annotations' %
+                STANDARD_ANNOTATIONS.keys()
+            )
+        else:
+            # get a list of all of the mapped annotations
+            self.annotation_filter = {
```

Comment:
Agreed. I think it's fine as is. You could probably write it as a map, but the simplest thing I can come up with is:

```python
self.annotation_filter = set()
map(lambda include_name: self.annotation_filter.update(STANDARD_ANNOTATIONS[include_name]), include)
```
which is considerably worse. I mean, you could also just do:
```python
self.annotations = set()
for include_name in include:
    self.annotations.update(STANDARD_ANNOTATIONS[include_name])
```

This has the nice property of not allowing `include` values that aren't in `STANDARD_ANNOTATIONS`, and is way more readable. Doesn't feel as pythonic, though.